cron/cron-daily.pl

```diff
@@ -80,7 +80,7 @@ sub read_errorlog {
     close LOG;
     report "\nErrorlog contains $errorlines lines today\n";
   } else {
-    warn "error opening $PAUSE::Config->{HTTP_ERRORLOG}: $!";
+    $Logger->log("error opening $PAUSE::Config->{HTTP_ERRORLOG}: $!");
```

Comment:
if this is an error, should it be fatal?

Reply:
I think the whole premise of this code is kinda wonky, so I'm gonna not touch it beyond changing its logging.  "Send me the day's web error logs" is odd.